src/persistence/user/cms_user_storage_i.e

```diff
@@ -56,6 +56,20 @@ feature -- Access
 			password: Result /= Void implies (Result.hashed_password /= Void and Result.password = Void)
 		end
 
+	user_by_activation_token (a_token: READABLE_STRING_32): detachable CMS_USER
```

Comment:
I guess this feature and the other should be in their own class in the related module.